### PR TITLE
udp_proxy: upgrade to not alpha

### PR DESCRIPTION
**Pull request by mattklein123**
This extension is used in production and we should treat it as such.

Risk Level: Low
Testing: Existing tests
Docs Changes: Updated
Release Notes: Updated

### CODE REVIEW

File: api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.udp.udp_proxy.v3";
+option java_outer_classname = "UdpProxyProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = NEXT_MAJOR_VERSION_CANDIDATE;
```
Comment (htuch) [inline api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto:14]: I think this should be active and the v2alpha now becomes frozen. Can you try manually to make those changes and see what the tooling then does?

Comment (mattklein123) [reply api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto]: I think it worked, PTAL. (I think coverage will still be broken though until they fix the disk issue.)

**Comment by mattklein123** [comment]
@htuch coverage actually passed here, not sure why it's marked as failure. PTAL.

**Pull request merged**